wrappers/javascript/nodejs/package.json

```diff
@@ -47,10 +48,18 @@
     "typescript": "4.5.5"
   },
   "dependencies": {
-    "indy-vdr-shared": "*",
+    "@mapbox/node-pre-gyp": "^1.0.10",
     "ffi-napi": "^4.0.3",
+    "indy-vdr-shared": "0.1.0",
```

Comment:
Always unsure about this, when using this locally, will this pick up the version inside your project or will it use the `0.1.0` from npm?

Reply:
Yes this will work locally if the version matches the version of the package in the local workspace. Need to do this as otherwise yarn won't correctly increase the version for other packages.